### DISCUSSION TITLE
Laragon 6: Can't login to MySQL

### DISCUSSION THREAD

**Discussion by nguyentrongchinh1997** [leokhoa/laragon#527] [Q&A]
After install phpMyadmin in /etc/apps. When access to localhost/phpmyadmin, can't login with username: root and no password

**Comment by DebZakari** [comment]
I also encountered this problem, and my fix was quite simple. 

It was a company PC and a previous user used Laragon as well, and they had a password. 

So, I just located the MySQL "my.ini" file and looked at the password. Login using that password and all is well.
